admin/create-theme/theme-form.php

```diff
@@ -223,7 +223,10 @@ public static function create_admin_form_page() {
 	}
 
 	public static function form_script() {
```

Comment:
This is a great catch; it was frustrating me quite a bit, as it wasn't working with the extra `/` present.

It's working for me now.